### PR TITLE
Footer #2

### LINKED ISSUE #1: Complete the Landing Page

### LINKED ISSUE #2: Footer

### PR DESCRIPTION
**Pull request by MadhushaPrasad**
Footer #2

### CODE REVIEW

**Review by sithumonline — COMMENTED** [review]
Idea ???? Mv this to git ignore

File: pages/index.tsx [hunk 1]
```diff
@@ -15,6 +16,7 @@ const Homepage = () => {
       <MakeGoodThings />
       <WhatWeDo />
       <OurPartners />
+      <Footer/>
```
Comment (sithumonline) [inline pages/index.tsx:19]: Not in right aline

**Review by sithumonline — CHANGES_REQUESTED** [review]
LGTM BABABA

**Review by MadhushaPrasad — COMMENTED** [review]
done

**Review by MadhushaPrasad — COMMENTED** [review]
fixed

**Review by sithumonline — APPROVED** [review]

**Pull request merged**